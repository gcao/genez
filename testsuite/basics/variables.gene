#!/usr/bin/env gene run

# Test: Variable declaration and access
(var x 10)
(var y 20)
(print "x = " x)
(print "y = " y)

# Variable shadowing
(var x 30)
(print "x after shadowing = " x)