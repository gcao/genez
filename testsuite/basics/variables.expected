x =  10
y =  20
x after shadowing =  30